src/SixLabors.Fonts/Tables/Cff/Cff1Parser.cs

```diff
@@ -0,0 +1,1256 @@
+           // the offset operand of the Subrs  operator in the Private DICT.
+           // A font without local subrs has no Subrs operator in the Private DICT.
+
+           // Global subrs are stored in an INDEX structure which follows the
+           // String INDEX. A FontSet without any global subrs is represented
+           // by an empty Global Subrs INDEX.
+           => this.ReadSubrBuffer(reader);
+
+        private byte[][] ReadLocalSubrs(BigEndianBinaryReader reader) => this.ReadSubrBuffer(reader);
+
+        private void ReadEncodings(BigEndianBinaryReader reader)
+        {
+            // Encoding data is located via the offset operand to the
+            // Encoding operator in the Top DICT.
+
+            // Only one Encoding operator can be
+            // specified per font except for CIDFonts which specify no
+            // encoding.
+
+            // A glyph’s encoding is specified by a 1 - byte code that
+            // permits values in the range 0 - 255.
+
+            // Each encoding is described by a format-type identifier byte
+            // followed by format-specific data.Two formats are currently
+            // defined as specified in Tables 11(Format 0) and 12(Format 1).
+            byte format = reader.ReadByte();
+            switch (format)
+            {
+                default:
+#if DEBUG
+                    System.Diagnostics.Debug.WriteLine("cff_parser_read_encodings:" + format);
+#endif
+                    break;
+                case 0:
+                    this.ReadFormat0Encoding(reader);
+                    break;
+                case 1:
+                    this.ReadFormat1Encoding(reader);
+                    break;
+            }
+
+            // TODO: ...
+        }
+
+        private void ReadCharsets(BigEndianBinaryReader reader, string[] stringIndex, CffGlyphData[] glyphs)
+        {
+            // Charset data is located via the offset operand to the
+            // charset operator in the Top DICT.
+
+            // Each charset is described by a format-
+            // type identifier byte followed by format-specific data.
+            // Three formats are currently defined as shown in Tables
+            // 17, 18, and 20.
+            reader.BaseStream.Position = this.offset + this.charsetOffset;
+            switch (reader.ReadByte())
+            {
+                default:
+                    throw new NotSupportedException();
+                case 0:
+                    this.ReadCharsetsFormat0(reader, stringIndex, glyphs);
+                    break;
+                case 1:
+                    this.ReadCharsetsFormat1(reader, stringIndex, glyphs);
+                    break;
+                case 2:
+                    this.ReadCharsetsFormat2(reader, stringIndex, glyphs);
+                    break;
+            }
+        }
+
+        private void ReadCharsetsFormat0(BigEndianBinaryReader reader, string[] stringIndex, CffGlyphData[] glyphs)
```

Comment:
`ReadCharsetsFormat0` and `ReadCharsetsFormat1` are not covered by tests according to the code coverage report. Would be good, if we have tests for those cases.

Reply:
I'll double check. The should be getting called.